src/Fixer/CastNotation/CastSpacesFixer.php

```diff
@@ -56,31 +71,42 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
-        static $insideCastSpaceReplaceMap = [
-            ' ' => '',
-            "\t" => '',
-            "\n" => '',
-            "\r" => '',
-            "\0" => '',
-            "\x0B" => '',
-        ];
-
         foreach ($tokens as $index => $token) {
             if ($token->isCast()) {
                 $tokens[$index] = new Token([
                     $token->getId(),
-                    strtr($token->getContent(), $insideCastSpaceReplaceMap),
+                    strtr($token->getContent(), self::INSIDE_CAST_SPACE_REPLACE_MAP),
                 ]);
 
-                // force single whitespace after cast token:
-                if ($tokens[$index + 1]->isWhitespace(" \t")) {
-                    // - if next token is whitespaces that contains only spaces and tabs - override next token with single space
-                    $tokens[$index + 1] = new Token([T_WHITESPACE, ' ']);
-                } elseif (!$tokens[$index + 1]->isWhitespace()) {
-                    // - if next token is not whitespaces that contains spaces, tabs and new lines - append single space to current token
-                    $tokens->insertAt($index + 1, new Token([T_WHITESPACE, ' ']));
+                if ($this->configuration['space'] === 'single') {
+                    // force single whitespace after cast token:
+                    if ($tokens[$index + 1]->isWhitespace(" \t")) {
```

Comment:
There are three `if` nested levels and a `else` now. Can you rewrite those using early `continue`?